test/e2e/channel_defaults_webhook_test.go

```diff
@@ -181,5 +181,5 @@ func webhookObservedUpdate(ch *messagingv1.Channel) bool {
 }
 
 func webhookObservedUpdateFromDeliverySpec(d *eventingduck.DeliverySpec) bool {
-	return d.BackoffDelay != nil && d.Retry != nil && d.BackoffPolicy != nil
+	return d != nil && d.BackoffDelay != nil && d.Retry != nil && d.BackoffPolicy != nil
```

Comment:
Just curious, why did this code work before?

Reply:
Because it's a race condition, it's not always nil.